completions/ssh

```diff
@@ -455,17 +457,30 @@ _comp_xfunc_ssh_scp_remote_files()
     if [[ ${1-} == -d ]]; then
```

Comment:
I think it would be cleaner to switch only the replacement text specified to `sed`. For example, we may define a variable in the following way and pass the value to `sed`.

```suggestion
    local escape_replacement='\\&'
    if [[ $overescape == overescape ]]; then
        escape_replacement='\\\\\\&'
    fi
    if [[ ${1-} == -d ]]; then
```

Reply:
Yes, that's indeed much better.